gamebench_api_client/models/dataframes/generic/abstract_generic.py

```diff
@@ -21,6 +22,7 @@ def __init__(self, **request_parameters):
                 build and send a response.
         """
 
+        self.authenticator = Authenticator(**request_parameters)
```

Comment:
Let's fetch the parameters for Authenticator's init from settings.

Reply:
Thank you, I'll change that.

Reply:
👍